GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -312,7 +311,18 @@ public void beforeDrawEyes() {
                 getMainScene().addSceneObject(mSplashScreen);
             }
 
-            mScript.onInit(GVRViewManager.this);
+            try {
+                mScript.onInit(GVRViewManager.this);
+            } catch (Throwable t) {
+                t.printStackTrace();
```

Comment:
Yep. I think I made a mistake. I will output the stack trace with tag System.err.